src/main/java/org/jenkins/tools/test/PluginCompatTesterCli.java

```diff
@@ -47,6 +48,14 @@
  */
 public class PluginCompatTesterCli {
 
+    static {
+        String configFile = System.getProperty("java.util.logging.config.file");
+        String configClass = System.getProperty("java.util.logging.config.class");
+        if (configClass == null && configFile == null) {
+            new LoggingConfiguration();
+        }
+    }
```

Comment:
This is needed to get production to use our custom logging configuration (I tried to find a way to add a `MANIFEST.MF` entry to do this, but there was none). For tests this is enabled by adding `-Djava.util.logging.config.class=org.jenkins.tools.test.logging.LoggingConfiguration` in the Surefire configuration.